### PR TITLE
Completion a tag in the post textarea and the title text field.

### LINKED ISSUE #107: Completion a tag in the post textarea and the title text field.
Completion a tag in the post textarea.

---

投稿テキストエリアとタイトルテキストフィールドで Tag の補完機能を実装する。
"# + 任意の一文字入力"で、S-TIP に tags の問い合わせを実施し、候補をリストアップして、補完する。

### PR DESCRIPTION
**Pull request by tkhsknsk**
#107

### CODE REVIEW

File: src/search/templates/search/search.html [hunk 1]
```diff
@@ -86,7 +87,8 @@ <h3 class="panel-title">{% trans 'Latest posts' %}</h3>
                 <div class="row compose-content-div">
               	  <div class="compose-content-textarea-div col-xs-8">
 		              <label for="compose-content">{% trans 'Content' %}:</label><br/>
-		              <textarea class="form-control compose-content" rows="10" name="post" id="compose-content"></textarea>
+		              <textarea class="form-control compose-content" rows="10" name="post" id="compose-content" style="display: block"></textarea>
```
Comment (ta-hiroki) [inline src/search/templates/search/search.html]: Why are you `style="display: block"` the textarea?

Comment (tkhsknsk) [reply src/search/templates/search/search.html]: This is the required code, as written in ISSUE below.

https://github.com/s-tip/stip-common/pull/73#discussion_r517782927

Comment (tkhsknsk) [reply src/search/templates/search/search.html]: It was deleted.

**Review by ta-hiroki — APPROVED** [review]

**Review by uchimrina — APPROVED** [review]

**Review by stmtstk — CHANGES_REQUESTED** [review]
Could you change the function name `start` (i.e. `start_suggest`)?
The developer may not know what starts with that function name.

**Review by stmtstk — APPROVED** [review]

**Pull request merged**